src/content/docs/tutorial-infrastructure-monitoring/explore-host-data.mdx

```diff
@@ -0,0 +1,124 @@
+
+* A few Springboot apps running in JVMs
+* Some microservices written in Ruby
+* Some microservices written in Go 
+
+In this case, the guided install would recommend:
+
+* Our infrastructure agent for your Apache VMs
+* The Java APM agent for your Springboot app
+* Additional APM installs for your Ruby and Go microservices
+
+You can choose to install the infrastructure agent only, or some combination of these recommendations. To get the most out of New Relic and this tutorial, we recommend you instrument anything you want to monitor.
+
+<Callout variant="important">
+While New Relic offers a full suite of tools and integrations to monitor Cloud infrastructure, Kubernetes, and other infra software, this tutorial is specific to data reported out of the box by our infra agent.
+</Callout>
+        
+    </Step>
+    <Step>
+
+Once installed, you can find your infrastructure data in our UI. Go to **[one.newrelic.com > All Capabilities > Infrastructure > Hosts](https://one.newrelic.com/infra/infrastructure/hosts)**. You're now looking at a summary page about your instrumented hosts. 
+
+<img
+    title="Infrastructure UI hosts summary page"
+    alt="A screenshot of the first page in the infrastructure UI when you go from All Capabilities to Infrastructure. It displays data about your hosts."
+    src={hostSummaryPage}
+/>
+
+Each host page is separated into three sections: filter actions, metric graphs, and a table. The data represents changes in your golden metrics over time — that is, changes in host CPU usage, memory usage, disk usage, and disk utilization. 
+
+    </Step>
+    <Step>
+    ### Toggle the time picker to adjust host metrics
+
+You can toggle the time picker to adjust the scope of your data. This is best used when making comparisons across time, like if you want to understand when a change first occurred. 
+
+Let's look at these metrics graphs toggled between 3 hours ago and 6 hours ago.
+
+<SideBySide>
+    <Side>
+<img
+    title="Tiles and graphs set to a 3 hour time parameter"
+    alt="A screenshot displaying two metrics graphs and tiles from the past 6 hours."
+    src={metrics1}
+
+/>    </Side>
+    <Side>
+<img
+    title="Tiles and graphs set to a 6 hour time parameter"
+    alt="A screenshot displaying two metrics graphs and tiles from the past 6 hours."
+    src={metrics2}
+/>    
+</Side>
+</SideBySide>
+
+* If there's been an unexpected increase in load, the **Events** tile will display that behavior with either too many or too few expected events. When you compare between two time parameters, you can get a better sense of anomalous behavior.   
+* The **Alerts** tile displays the number of hosts that have reached a critical or warning threshold. It can indicate whether you've found the parameter where an incident first appeared. 
```

Comment:
>It can indicate whether you've found the parameter where an incident first appeared.

I don't understand this point. What parameter are we referring to?